packages/react-core/src/components/Switch/examples/Switch.md

```diff
@@ -170,9 +170,29 @@ import { Switch } from '@patternfly/react-core';
 import React from 'react';
 import { Switch } from '@patternfly/react-core';
 
-<React.Fragment>
-  <Switch id="uncontrolled-switch-on" label="Message when on" labelOff="Message when off" defaultChecked={false} />
-  <br />
-  <Switch id="uncontrolled-no-label-switch-on" aria-label="Message when on" defaultChecked={true} />
-</React.Fragment>;
+class UncontrolledSwitch extends React.Component {
```

Comment:
I missed that this was supposed to be an uncontrolled example. There shouldn't be any state updates that affect the Switch. What is the purpose of tracking the checked state here?

Reply:
I guess I'm still not sure what the purpose of uncontrolled would be, but I am added tracking the state to show a consumer a way they would be able to track it, and passed the value back into the component as a way for the component to update the aria-labeledby value.

Reply:
I think it might make a little more sense or feel a little more intuitive to have a console statement in the onChange function in the example rather than updating an unused state variable.

And I also would have expected that the `defaultChecked` would be a hard coded true (for the case of the example). If the switch should be checked on page load, then they could use the `defaultChecked` prop. 

I don't think a new `ref` is necessary - as long as the `label` and `labeloff` props are updating the `aria-label` appropriately. Are we imagining the consumer needs another way to further update the aria-label?

Reply:
@nicolethoen - label and labeloff will switch fine, but the aria-labelledby will not update along with it. this is why i was updating the defaultChecked value.. but perhaps i could find another way, like onchange or ref within the switch component?

Reply:
@nicolethoen with Dallas' current solution, the `aria-labelledby` gets updated as I expected based on the `isChecked` prop value. With an uncontrolled variant, though, it gets trickier to update the `label`/`labelOff` or `aria-label` without that `isChecked` prop (we wouldn't necessarily need to update the `aria-labelledby` if we only pass the `label` prop and not `labelOff` since they'd both end up with the same text content that way).

My thoughts for adding a `ref` prop were basically to allow consumers to make updates/validations elsewhere based on the input value from the DOM since they wouldn't have that `isChecked` state to do so. This isn't something we do in other uncontrolled variants, though, so perhaps would be a separate discussion.

An alternative could be to just advise consumers to only use a static label/aria-label for an uncontrolled variant (rather than an on and off label like the other examples have) for now.